tools/distrib/yapf_code.sh

```diff
@@ -53,7 +53,7 @@ for dir in $DIRS; do
   tempdir=`mktemp -d`
   cp -RT $dir $tempdir
   $PYTHON -m yapf -i -r -p $exclusion_args $dir
-  if ! diff -rq $dir $tempdir; then
+  if ! diff -r $dir $tempdir; then
```

Comment:
I agree with your reasoning about successive steps of a test. Here all the directories seem like one single test step, like a single-instruction-multiple-data kind of thing. :smile: